$MirOS: ports/audio/libogg/patches/patch-configure_in,v 1.2 2005/11/03 19:23:34 bsiegert Exp $
$OpenBSD: patch-configure,v 1.6 2003/12/14 00:52:22 naddy Exp $
--- configure.in.orig	Mon Nov 28 04:25:45 2005
+++ configure.in	Wed Dec 28 15:51:59 2005
@@ -37,7 +37,7 @@ if test -z "$GCC"; then
 		;;
         *)
                 DEBUG="-g"
-                CFLAGS="-O"
+                CFLAGS=""
                 PROFILE="-g -p" 
 		;;
         esac
@@ -60,8 +60,8 @@ else
 		;;
         *)
                 DEBUG="-g -Wall -fsigned-char"
-                CFLAGS="-O20 -fsigned-char"
-                PROFILE="-O20 -g -pg -fsigned-char" 
+                CFLAGS="-fsigned-char"
+                PROFILE="-g -pg -fsigned-char" 
 		;;
         esac
 fi
